addons/game.controller.genesis.3button/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.genesis.3button"
+        name="Genesis Controller (3-Button)"
+        version="1.0.0"
+        provider-name="Team Kodi">
+    <extension point="kodi.game.controller" library="resources/layout.xml"/>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <assets>
+            <icon>resources/icon.png</icon>
+        </assets>
+        <summary lang="en_GB">Sega Genesis controller (3-Button)</summary>
```

Comment:
Button should probably be "Sentence case." for summary

Reply:
I was trying to follow the pattern of other controllers here:
https://github.com/kodi-game/controller-topology-project/blob/1a90e1241206b564d0fad7bb64a2091729ce9d30/addons/game.controller.joystick.2button/addon.xml#L15

So should I keep it? Or do you want me to change this one and you will fix the rest? Or do we make another PR to fix this for all controllers at once?

Reply:
Other candidates:
https://github.com/kodi-game/controller-topology-project/blob/1a90e1241206b564d0fad7bb64a2091729ce9d30/addons/game.controller.joystick.4button/resources/language/resource.language.en_gb/strings.po#L20
https://github.com/kodi-game/controller-topology-project/blob/1a90e1241206b564d0fad7bb64a2091729ce9d30/addons/game.controller.konami.justifier.player2/resources/language/resource.language.en_gb/strings.po#L20
https://github.com/kodi-game/controller-topology-project/blob/1a90e1241206b564d0fad7bb64a2091729ce9d30/addons/game.controller.ps.guncon.western/resources/language/resource.language.en_gb/strings.po#L20
https://github.com/kodi-game/controller-topology-project/blob/1a90e1241206b564d0fad7bb64a2091729ce9d30/addons/game.controller.saturn.3d.western/resources/language/resource.language.en_gb/strings.po#L20
https://github.com/kodi-game/controller-topology-project/blob/1a90e1241206b564d0fad7bb64a2091729ce9d30/addons/game.controller.saturn.virtua.gun.eu/resources/language/resource.language.en_gb/strings.po#L20

So I'd keep it.

Reply:
Generally, a "name" field should be "Title Case" and "summary" and "description" fields should be "Sentence case." However, "name" isn't translated, while summary is, so I think we use summary as a name in the GUI. In that case, summary should also be title case.

I think the question is, what looks better in the UI. Check the port dialog and controller mapper, and if you think that Title Case looks better than Sentence case then we'll move everything to that. I suspect it does, however most of the rest of Kodi is Sentence case so maybe using that for our summaries is better.

Reply:
We do have another translated string (#30000) that is being used in the port dialog and the controller dialog:
It is "Genesis (3-Button)" in this case.

When I search the UI for the summary I can only see it in the addon information dialog, nowhere else.
So the sentence case for the summary is fine, there is no problem with that.

The question is only about the content in the brackets in summary. Currently all controllers, which have anything in brackets in the summary, have capital letter.

I'm not native english speaker, so I'm not sure whether e.g. "_(european)_" is right (it is in my language). If that needs the capital, then only the "(Player 2)" and the "(4-Button)" are in question here.